src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -293,8 +368,69 @@ private TextSpan GetRegionSpanForReference(SourceText sourceText, TextSpan refer
                     sourceText.Lines[lastLineNumber].End);
             }
 
+            private void UpdateCustomColumnsVisibility(MultiDictionary<string, string> customData)
+            {
+                // Check if we have any custom reference data to display.
+                if (customData.Count == 0)
```

Comment:
can this be null?  

also, as an aside, can we make sure hte common case of 0 values is cheap?  i.e. they all point at the same empty dictionary?  if you already did this, great!